docs/usage/templates.md

```diff
@@ -1,4 +1,4 @@
----
+Encode URI---
```

Comment:
Can you explain why you're making this change? It's the first time I'm seeing something like this?

It also looks like the current change will mess up the front matter.